nvflare/dashboard/application/models.py

```diff
@@ -98,11 +118,13 @@ class User(CommonMixin, db.Model):
     role = db.relationship("Role", backref="users")
     organization_id = db.Column(db.Integer, db.ForeignKey("organization.id"), nullable=False)
     organization = db.relationship("Organization", backref="users")
+    props = db.Column(db.String(1048), default="")  # additional properties - JSON string
```

Comment:
should we do 2048 as others?

Reply:
This is for admin users. We don't expect many props for users.